Library/Homebrew/build.rb

```diff
@@ -103,6 +103,13 @@ def install
       end
     end
 
+    old_tmpdir = ENV["TMPDIR"]
+    old_temp = ENV["TEMP"]
+    old_tmp = ENV["TMP"]
+    ENV["TMPDIR"] = HOMEBREW_TEMP
+    ENV["TEMP"] = HOMEBREW_TEMP
+    ENV["TMP"] = HOMEBREW_TEMP
```

Comment:
Yeah, they can be chained, but AFAIK some people detest this style and I've seen style guides (in other languages) that explicitly advise against chaining assignments before. I guess Homebrew maintainers don't mind?